src/App.tsx

```diff
@@ -12,7 +12,7 @@ export const App: React.FC<{}> = () => (
   <StylesProvider injectFirst>
     <MuiThemeProvider theme={theme}>
       <ThemeProvider theme={theme}>
-        <Router>
+        <Router basename={`${process.env.PUBLIC_URL}`}>
```

Comment:
We need to add Github repo name (`tech-transition`) to our paths to take into account how Github constructs pages urls. Otherwise, the routing will not work after deployment.